docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,120 @@
+                </path/to/new/dir>: {make: true}
+                </path/to/file/with/text>: {text: "Some text"}
+                </path/to/mutable/file/with/default/text>: {text: FIXME, mutable: true}
+                </path/to/temporary/content>: {until: mutate}
+
+            # (opt) Mutation scripts, to allow for the reproduction of maintainer scripts,
+            # based on Starlark (https://github.com/canonical/starlark)
+            mutate: |
+                ...
+
+Find the dependencies of your package
+-------------------------------------
+
+Find the dependencies of the package for which you want to create a new slice definitions (``openssl`` in this guide) with this command:
+
+.. code-block:: sh
+
+    $ apt show openssl
+
+.. code-block:: text
+
+    package: openssl
+    Version: 3.0.2-0ubuntu1.7
+    Origin: Ubuntu
+    Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
+    Original-Maintainer: Debian OpenSSL Team <pkg-openssl-devel@alioth-lists.debian.net>
+    Bugs: https://bugs.launchpad.net/ubuntu/+filebug
+    Depends: libc6 (>= 2.34), libssl3 (>= 3.0.2-0ubuntu1.2)
+
+From the above output, you can confirm that ``openssl`` **depends on** ``libc6`` and ``libssl3``. So when creating your slice definitions file for OpenSSL, you will need to remember to include those packages' slices as a dependency as well, whenever needed. Let's do that in the following section.
+
+Create your slice definition
+----------------------------
+
+You now have everything needed to actually define the OpenSSL slice that will install the content you are looking to have in the final ROCK.Since you are looking to install just the OpenSSL binaries from the ``openssl`` package, what about naming this new slice **bins**? Let's go for it:
+
+#. **What is the name of your slice definitions file?** It is a YAML file called ``openssl.yaml``
+#. **What package name should be defined inside this file?** The package name is ``openssl``
+#. **What is your slice name?** It should be called ``bins``
+#. **What contents do you need from the OpenSSL package?** Just the binaries - ``/usr/bin/c_rehash`` and ``/usr/bin/openssl``
+#. **Does your slice depend on any other package slice?** Yes, OpenSSL depends on ``libc6`` and ``libssl3``
+
+   * **Do these two packages have slice definitions files upstream?** Yes, there is already a slice definitions file for `libc6 <https://github.com/canonical/chisel-releases/blob/ubuntu-22.04/slices/libc6.yaml>`_ and another one for `libssl3 <https://github.com/canonical/chisel-releases/blob/ubuntu-22.04/slices/libssl3.yaml>`_ . If these dependencies were not present in the upstream Chisel release, you will also need to create their corresponding slice definitions
+   * **Which slices do you depend on then?** Since you only want the OpenSSL binaries, you might only need the libraries from ``libc6`` and ``libssl3``, as well as the configuration files from ``libc6`` and ``openssl`` itself.
```

Comment:
```suggestion
   * **Which slices do you depend on then?** Since you only want the OpenSSL binaries, you might only need the libraries from ``libc6`` and ``libssl3``, as well as the configuration files from ``libc6`` and ``openssl`` themselves.
```